docs/antctl.md

```diff
@@ -476,6 +476,9 @@ $ antctl traceflow -S ns1/pod1 -D 123.123.123.123
 $ antctl traceflow -S pod1 -D ns1/svc1 -f tcp,tcp_dst=80
 # Start a Traceflow from pod1 to pod2, with a UDP packet to destination port 1234
 $ antctl traceflow -S pod1 -D pod2 -f udp,udp_dst=1234
+# Start a Traceflow from pod1 to pod2, with a TCP SYN packet to destination port 1234
+# tcp_flags will be 2 by default when it's not provided.
+$ antctl traceflow -S pod1 -D pod2 -f tcp,tcp_dst=1234,tcp_flags=2
```

Comment:
Not sure why we are adding this example at all, since the flags are now being set by default. Explicitly using `tcp_flags=2` is a bit misleading?

Reply:
Removed